tests/unit/test.program.js

```diff
@@ -480,21 +507,154 @@ describe('program.main', () => {
       return configObject;
     });
 
-    return execProgram(
+    await execProgram(
       ['lint', '--config', 'path/to/web-ext-config.js'],
       {
         commands: fakeCommands,
         runOptions: {
           loadJSConfigFile: fakeLoadJSConfigFile,
         },
-      })
-      .then(() => {
-        const options = fakeCommands.lint.firstCall.args[0];
-        // This makes sure that the config object was applied
-        // to the lint command options.
-        assert.equal(
-          options.selfHosted, configObject.lint.selfHosted);
-      });
+      }
+    );
+
+    const options = fakeCommands.lint.firstCall.args[0];
+    // This makes sure that the config object was applied
+    // to the lint command options.
+    assert.equal(
+      options.selfHosted, configObject.lint.selfHosted);
+  });
+
+  it('discovers config files', async () => {
+    const fakeCommands = fake(commands, {
+      lint: () => Promise.resolve(),
+    });
+    const configObject = {
+      lint: {
+        selfHosted: true,
+      },
+    };
+    // Instead of loading/parsing a real file, just return an object.
+    const fakeLoadJSConfigFile = sinon.spy(() => {
```

Comment:
This is defined as a `sinon.spy`, we could use it to assert that it is called with the expected filename, am I right?

Reply:
Good catch, fixed